main.go

```diff
@@ -172,6 +173,16 @@ func _main(ctx context.Context) error {
 			DBName:       cfg.PGDBName,
 			QueryTimeout: cfg.PGQueryTimeout,
 		}
+		// Replace {netname} with curnet
+		if strings.Contains(dbi.DBName, "{netname}") {
+		  reg, err := regexp.Compile("[^a-zA-Z]+")
+		  if err != nil {
+		    return err
+		  }
+		  
+		  var networkName = reg.ReplaceAllString(curnet.String(), "");
```

Comment:
For consistency, when defining a variable, use the syntax that omits var (`networkName := ...`).  Only when declaring with the intention of starting with the type's zero value do we do `var netName string`.